### PR TITLE
371 updates not alone and definition copy

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/B0ClLPYb/371-master-copy-for-beta-01

## Description

- Definition copy update
- Not alone copy update

@neontribe/sea-map

### CODE REVIEW

**Review by Nikomus — DISMISSED** [review]
Neat! 🍌

File: tests/DefinitionTest.elm [hunk 1]
```diff
@@ -33,7 +33,7 @@ suite =
                 \() ->
                     queryFromStyledHtml (view initModel)
                         |> Query.contains [ Html.text (t DefinitionTitle) ]
-            , test "Definition view has 6 nav links (1 introduction, 2 navigation, 3 emergency)" <|
+            , test "Definition view has 7 nav links (2 navigation, 4 emergency)" <|
```
Comment (katjam) [inline tests/DefinitionTest.elm]:
```suggestion
            , test "Definition view has 5 nav links (2 navigation, 3 emergency)" <|
```

File: tests/DefinitionTest.elm [hunk 2]
```diff
@@ -33,11 +33,11 @@ suite =
                 \() ->
                     queryFromStyledHtml (view initModel)
                         |> Query.contains [ Html.text (t DefinitionTitle) ]
-            , test "Definition view has 7 nav links (2 navigation, 4 emergency)" <|
+            , test "Definition view has 5 nav links (2 navigation, 3 EMERGENcy)" <|
```
Comment (Nikomus) [inline tests/DefinitionTest.elm]:
```suggestion
            , test "Definition view has 5 nav links (2 navigation, 3 emergency)" <|
```

**Review by Nikomus — APPROVED** [review]
GREat 🍌

**Pull request merged**